### PR TITLE
Fix linting errors caused by pylint upgrade

### PR DESCRIPTION
**Pull request by bhushanthakur93**

### CODE REVIEW

File: reconcile/saas_file_owners.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def collect_owners():
 
     # make owners suitable for json dump
     for k in owners:
-        owners[k] = list(owners[k])
+        owners[k] = list(owners[k])  # pylint: disable=modified-iterating-dict
```
Comment (fishi0x01) [inline reconcile/saas_file_owners.py:51]: Lets fix instead:

```
    # make owners suitable for json dump
    ans = {}
    for k, v in owners.items():
        ans[k] = list(v)

    return ans
```

Comment (fishi0x01) [reply reconcile/saas_file_owners.py:51]: Or better actually

```
 # make owners suitable for json dump
 return {k: list(v) for k, v in owners.items()}
```

Comment (bhushanthakur93) [reply reconcile/saas_file_owners.py]: Addressed.

**Review by fishi0x01 — APPROVED** [review]
Thanks for fixing this! :bow:

**Pull request merged**